app/src/main/java/com/mashup/mobalmobal/ui/profile/data/service/ProfileService.kt

```diff
@@ -0,0 +1,14 @@
+package com.mashup.mobalmobal.ui.profile.data.service
+
+import com.mashup.mobalmobal.ui.profile.data.dto.ProfileResultDto
+import io.reactivex.Single
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface ProfileService {
+    @GET("/users")
```

Comment:
```suggestion
    @GET("users")
```
매번 / 넣긴 힘드니 END_POINT에 /를 넣어주고 모든 Url 시작에는 / 빼시죵

Reply:
그러시죵!

Reply:
수정했어요~
커밋넘버 : 96064b